examples/README.md

```diff
@@ -48,20 +50,20 @@ During my run of `./security-setup` I was asked:
     ----> admin password
     Admin Password:
 
-
-At that point I had entered 'hardpass'
+which is the password used for Marathon.
 
 ### Other Clusters
 
-If you want to dive right into a cloud or an openstack cluster go through
+If you want to dive right into a cloud cluster go through
 [Getting Started](https://microservices-infrastructure.readthedocs.org/en/latest/getting_started/index.html)
-at the Documetation site.   You can then change the relevant parts of the following instructions.
+at the Documetation site. You can then change the relevant parts of the
+following instructions.
 
 ### Submitting Applications to The Marathon API
 
-While in a terminal, in this examples, directory enter:
+While in a terminal, in this examples directory enter:
 
-    curl -k -X POST -H "Content-Type: application/json" "https://admin:hardpass@192.168.242.55:8080/v2/apps" -d@"hello-world/hello-world.json"
+    $ curl -k -X POST -H "Content-Type: application/json" -u "admin:hardpass" -d@"hello-world/hello-world.json" "https://192.168.242.55:8080/v2/apps"
```

Comment:
these examples are using the "old" vagrant ip address. should we update these to use the new default?